src/Widgets/SizeLabel.vala

```diff
@@ -37,7 +37,6 @@ public class AppCenter.Widgets.SizeLabel : Gtk.Grid {
         );
         size_label = new Gtk.Label (null);
         size_label.hexpand = true;
-        size_label.vexpand = true;
```

Comment:
This was vexpanded to make it centered alongside the button here:

![image](https://user-images.githubusercontent.com/611168/61970312-bf8c2280-af99-11e9-8f86-7a361f5f769f.png)

But that could also be applied to it just on the updates view?

Reply:
Ah good catch, I'll fix that